src/PaymentGateways/Gateways/Stripe/Actions/EnqueueStripeFormBuilderScripts.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Give\PaymentGateways\Gateways\Stripe\Actions;
+
+/**
+ * Class EnqueueStripeFormBuilderScripts
+ *
+ * @since 3.0.0
+ */
+class EnqueueStripeFormBuilderScripts
+{
+
+    /**
+     * Enqueues the Stripe scripts and styles for the Form Builder.
+     *
+     * @return void
+     */
+    public function __invoke()
```

Comment:
Same comment here about moving these actions to `Gateways\Stripe\StripePaymentElementGateway`.  It looks specific to payment element

Reply:
Sure

Reply:
Resolved: https://github.com/impress-org/givewp/pull/6903/commits/dc77ec96213d780460efe0875f9502707fcf6a21